portal-impl/src/com/liferay/portal/service/impl/RoleLocalServiceImpl.java

```diff
@@ -682,6 +682,15 @@ else if (group.isLayout() || group.isLayoutSetPrototype() ||
 	}
 
 	@Override
+	public int getRegularRoleUsersGroupsCount(Role role) {
```

Comment:
I think this method should only receive the roleId, that way if the client only has the id doesn't have to retrieve the role obj from the db just to invoke this method.